prac_06/guitars.py

```diff
@@ -0,0 +1,41 @@
+"""
+guitars.py
+Estimated Time: 15 minutes
+Start Time: 8:10 AM
+End Time: 8:25 AM, 15 minutes
+
+This program allows the user to enter details of their guitars, stores them
+in a list, and displays them with vintage labeling if applicable.
+"""
+
+from prac_06.guitar import Guitar
+
+def main():
+    """Prompt user to input guitar details and display all entered guitars."""
+    guitars = []
+    print("My guitars!")
+
+    # Get the first input outside the loop to start the condition
+    name = input("Name: ")
+
+    while name:  # Continue as long as name is not blank
```

Comment:
This while loop won't work as it just says "while name" and doesn't have a condition.
For it to works, it needs to be:
while name != ""

Reply:
Thanks for your feedback but it will work mate as it says to run the loop until it exists